<launch>
  <let name="tf_prefix" value="$(eval '\'$(env ROS_NAMESPACE not_defined)\' if \'$(env ROS_NAMESPACE not_defined)\' != \'not_defined\' else \'\'')"/>
  <arg if="$(env IS_ROBOT false)" name="taskset" default="taskset -c 3"/>
  <arg unless="$(env IS_ROBOT false)" name="taskset" default=""/>

  <node pkg="bitbots_dynamic_kick" exec="KickNode" output="screen" launch-prefix="$(var taskset)">
    <param from="$(find-pkg-share bitbots_dynamic_kick)/config/kick_config.yaml" />

    <!--param name="base_link_frame" value="$(var tf_prefix)base_link"/>
    <param name="base_footprint_frame" value="$(var tf_prefix)base_footprint"/>
    <param name="r_sole_frame" value="$(var tf_prefix)r_sole"/>
    <param name="l_sole_frame" value="$(var tf_prefix)l_sole"/-->
  </node>
</launch>
